care_o_bot_simulation/package.xml

```diff
@@ -10,7 +10,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <run_depend>cob_simulation</run_depend>
+  <exec_depend>cob_simulation</exec_depend>
```

Comment:
Does this really install all packages required for simulation?

In fact, in case dependencies are complete and correct, for `care_o_bot_simullation` only an exec_depend to `cob_bringup_sim` should be required.
Same for `care_o_bot_robot`, an exec_depend to `cob_bringup` should be sufficient....

Let's discuss this once new releases have been triggered for the other packages....as this one is going to be released last anyway...